app/controllers/concerto_plugins_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
         restarted = restart_webserver()
       end
       if restarted
-        flash[:notice] = t(:plugin_created)
+        flash[:notice] = t(:was_created, name: @concerto_plugin.name, theobj: t(:plugin))
```

Comment:
Coders love to reduce duplication, but this does not work for natural languages. It looks like this duplication is a lot of unnecessary work for translators, but it makes their life actually easier. They can e.g. use Translation memory to speed up the process and still have the ability to respect some language specifics when translating. And it is much easier to translate a real sentence than something like ```The %{name} %{theobj} was successfully created.``` where the translator needs to account for all future possible words that might be filled in there. 

And this simple string, can't be translated to a pretty similar language to english, which happens to be German. There we would have to distinguish between male and female ```%{name}```and use different articles and can't just use ```The```. So in german it should be ```%{article} %{name} %{theobj} was successfully created.```. 

When you are trying to translate this into even more different languages, that have complexity different grammatical structures and concepts, it is impossible to correctly translate this.  
You will end up with ```%{article} %{name} %{theobj} %{temporalverb} %{adjective} %{verb}.```